src/imgui_impl_sdl_gl3.cc

```diff
@@ -39,13 +39,14 @@
 #define _CRT_SECURE_NO_WARNINGS
 #endif
 
-#include "imgui.h"
+#include "imgui/imgui.h"
```

Comment:
Squash this with previous.

Reply:
previous is just a straight copy paste of their stuff. Doesn't build or function.